app/views/additional_payments/claims/check_your_answers_part_one.html.erb

```diff
@@ -1,23 +1,15 @@
-<% content_for(:page_title, page_title(t("additional_payments.check_your_answers.part_one.primary_heading"), journey: current_journey_routing_name, show_error: current_claim.errors.any?)) %>
-<% path_for_form = current_claim.persisted? ? claim_path(current_journey_routing_name) : claims_path(current_journey_routing_name) %>
+<% content_for(:page_title, page_title(t("additional_payments.check_your_answers.part_one.primary_heading"), journey: current_journey_routing_name)) %>
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <%= render("shared/error_summary", instance: current_claim) if current_claim.errors.any? %>
-
     <h1 class="govuk-heading-l">
       <%= t("additional_payments.check_your_answers.part_one.primary_heading") %>
     </h1>
 
     <%= render partial: "claims/check_your_answers_section", locals: {heading: "Eligibility details", answers: journey.answers_for_claim(current_claim).eligibility_answers} %>
 
-    <%= form_for current_claim, url: path_for_form  do |form| %>
-
-      <p class="govuk-body"><%= t("additional_payments.check_your_answers.part_one.confirmation_notice") %></p>
+    <p class="govuk-body"><%= t("additional_payments.check_your_answers.part_one.confirmation_notice") %></p>
 
-      <div class="govuk-form-group">
-        <%= form.submit "Continue", class: "govuk-button", data: {module: "govuk-button"} %>
-      </div>
-    <% end %>
+    <%= button_to "Continue", claim_path(current_journey_routing_name), method: :patch, class: "govuk-button" %>
```

Comment:
Without pulling apart the ClaimsController too much, the other option is to create an empty form:

```ruby
module Journeys
  module AdditionalPaymentsForTeaching
    class CheckYourAnswersPartOne < Form
      def save
        true
      end
    end
  end
end
```

so that we fall into the new logic in the controller:

https://github.com/DFE-Digital/claim-additional-payments-for-teaching/blob/master/app/controllers/claims_controller.rb#L84-L93

but that doesn't seem necessary... maybe one to return to once all the rest of the forms are migrated?

Reply:
I think this approach is better than adding an empty form object + it's not hard to introduce one if we eventually need to do some checking on this page